cmd/kubeadm/app/util/version_test.go

```diff
@@ -238,9 +234,8 @@ func TestKubernetesIsCIVersion(t *testing.T) {
 		{"release/v1.0.0", false},
 		// CI builds
 		{"ci/latest-1", true},
-		{"ci-cross/latest", true},
+		{"ci/k8s-master", true},
```

Comment:
for reference:
https://github.com/kubernetes/test-infra/pull/14030
https://github.com/kubernetes/release/issues/854#issuecomment-524071615

`k8s-master` on the other hand is a label that i don't fully understand.
if you look here there are more `k8s-*` labels but i don't think we should support them in kubeadm for now.
https://console.cloud.google.com/storage/browser/kubernetes-release-dev/ci?pli=1

so i'm fine with the replacement this PR is doing.

Reply:
Yeah, I agree. k8s-master can be used to verify the cross builds for kubernetes when used with kubeadm.